src/components/BorderedList/BorderedList.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ListProps } from '@digdir/design-system-react';
+import { List } from '@digdir/design-system-react';
+import cn from 'classnames';
+import * as React from 'react';
+
+import classes from './BorderedList.module.css';
+
+export type BorderedListProps = {
+  /**
+   * Style of the border separating the items
+   */
+  borderStyle?: 'solid' | 'dashed';
```

Comment:
Because if you don't set it, it defaults to dashed - I just figured it would be nice not to _have_ to set it to dashed all the time, when that's by far the one we use the most anyway